app/javascript/lib/validation/expression-matrices-validation.js

```diff
@@ -353,13 +356,12 @@ function validateValuesAreNumeric(line, isLastLine, lineNum, dataObj) {
 }
 
 /**
-   * Validate the first line in the sparse Matrix begins with the string '%%MatrixMarket'
+   * Validate the first line in the sparse Matrix begins with '%%MatrixMarket'
    */
 function validateMTXHeaderLine(line) {
   const issues = []
-  const mtxHeader = line.slice(0, 14)
-  if (mtxHeader !== '%%MatrixMarket') {
-    const msg = `First line must begin with "%%MatrixMarket", not "${mtxHeader}"`
+  if (line[0] !== '%%MatrixMarket') {
+    const msg = `First line must begin with "%%MatrixMarket", not "${line}"`
```

Comment:
Blocker: we'll want to print the first element in `line`, not the whole array.

```suggestion
    const msg = `First line must begin with "%%MatrixMarket", not "${line[0]}"`
```